connectors/src/connectors/microsoft/temporal/spreadsheets.ts

```diff
@@ -134,9 +134,21 @@ async function processSheet(
   );
 
   // Content.text is guaranteed to be a 2D array with each row of the same length.
-  const rows: string[][] = content.value.text;
+  const rows: string[][] = content?.value?.text;
+  if (!rows) {
+    localLogger.info(
+      `[Spreadsheet] Cannot get any row from sheet ${worksheet.id} in document ${spreadsheet.id}.`
```

Comment:
Let's keep the same literal string for the message and provide arguments in the first parameter of the logger. This way, it's easier to search for logs.